addons/frag/XEH_postInit.sqf

```diff
@@ -1,32 +1,25 @@
 #include "script_component.hpp"
 
-if (isServer) then {
-    GVAR(lastFragTime) = -1;
-    [QGVAR(frag_eh), {_this call FUNC(frago);}] call CBA_fnc_addEventHandler;
-};
-
-["CBA_settingsInitialized", {
-    if (!GVAR(enabled)) exitWith {};
-
-    // Register fire event handler
-    ["ace_firedPlayer", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-    ["ace_firedNonPlayer", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-    ["ace_firedPlayerVehicle", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-    ["ace_firedNonPlayerVehicle", LINKFUNC(fired)] call CBA_fnc_addEventHandler;
-
-    addMissionEventHandler ["EachFrame", {call FUNC(masterPFH)}];
-}] call CBA_fnc_addEventHandler;
-
-// Cache for ammo type configs
-GVAR(cacheRoundsTypesToTrack) = [false] call CBA_fnc_createNamespace;
-
-
-// Debug stuff:
-
-#ifdef DRAW_FRAG_INFO
-[] call FUNC(dev_startTracing);
+[
+    "CBA_settingsInitialized",
+    {
+        if (isServer) then {
+            [] call FUNC(initBlackList);
+            [] call FUNC(initMaterialCache);
+        };
+
+#ifdef DEBUG_MODE_DRAW
+        if (hasInterface) then {
+            private _handle = [LINKFUNC(dev_drawTrace), 0] call CBA_fnc_addPerFrameHandler;
+            missionNamespace setVariable [QGVAR(dev_drawPFEH), _handle];
```

Comment:
```suggestion
            GVAR(dev_drawPFEH) = [LINKFUNC(dev_drawTrace), 0] call CBA_fnc_addPerFrameHandler;
```

Reply:
Fixed in https://github.com/acemod/ACE3/pull/9728/commits/3957a0751fad179896de9a433d8f5fd7b3a4b3eb